BlueprintUICommonControls/Sources/TextField.swift

```diff
@@ -10,6 +10,7 @@ public struct TextField: Element {
     public var onChange: ((String) -> Void)? = nil
     public var secure: Bool = false
     public var isEnabled: Bool = true
+    public var textAlignment: NSTextAlignment = .natural
```

Comment:
The default value on UITextField is `.left` – is it worth matching that? https://developer.apple.com/documentation/uikit/uitextfield/1619643-textalignment